File: cpp/examples/global_maps_registration.cpp
#include "registration_common.hpp"

int main() {
    auto set = examples::standard_fleet(
        {"test_bot_1", "test_bot_2"},
        horus::core::RobotType::WHEELED,
        {0.8, 0.55, 0.45},
        "wheeled");
    examples::add_world_maps(set.datavizs.front());
    return examples::register_or_fail(set);
}

File: rust/examples/global_maps_registration.rs
mod common;

use horus::{RobotDimensions, RobotType};

fn main() {
    let mut set = common::standard_fleet(
        &["test_bot_1", "test_bot_2"],
        RobotType::Wheeled,
        RobotDimensions::new(0.8, 0.55, 0.45),
        "wheeled",
    );
    common::add_world_maps(&mut set.datavizs[0]);
    common::register_or_fail(&mut set);
}
